### ISSUE TITLE
code of loss part may have some problem?

### ISSUE THREAD

**Issue by ousuixin** [Vious/LBAM_Pytorch#7]
hello, thank you for your excellent work and the open source code.
And I have some questions about InpaintingLoss.py:

https://github.com/Vious/LBAM_Pytorch/blob/98c2ae70486f4ba3ab86d4345e586e7841cfe343/loss/InpaintingLoss.py#L75

why not use  D_loss = D_fake - D_real + gp?
if D_real = D_real.mean().sum() * -1 and D_fake = D_fake.mean().sum() * 1
I think D_loss = D_fake + D_real + gp

and line 120
GLoss = holeLoss + validAreaLoss + prcLoss + styleLoss + 0.1 * D_fake 
should be 
GLoss = holeLoss + validAreaLoss + prcLoss + styleLoss - 0.1 * D_fake

thank you for your reply.

**Comment by liuyuyuil** [comment]
same confused ...

**Comment by heping0307** [comment]
Have you solved it?
